app/assets/javascripts/document-upload.js

```diff
@@ -99,9 +99,9 @@ var DocumentUpload = {
     disableSubmitButton: function() {
         $('.button').attr('disabled', 'disabled');
     },
-    addDataIndex: function() {
+    addDocumentRemoveLink: function() {
```

Comment:
Maybe give this function a more generic name? Like dunno... `amendLinkProperties`... :-/